certify-sunbird-insurance.properties

```diff
@@ -1,41 +1,43 @@
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential,VehicleInsuranceCredential
 
-## -------------------------------------- Authentication & Authorization -----------------------------------------------
+  ##---------------------------------Life Insurance Credential Configurations------------------------------------------------------
 
-mosip.certify.security.auth.post-urls={}
-mosip.certify.security.auth.put-urls={}
-mosip.certify.security.auth.get-urls={}
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/life-insurance-credential.json
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-id=did:schema:9ae73412-5783-4194-acec-6d004006660c
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-version=1.0.0
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
 
-mosip.certify.security.ignore-csrf-urls=${server.servlet.path}/actuator/**,${server.servlet.path}/error,\
-  ${server.servlet.path}/swagger-ui/**,${server.servlet.path}/v3/api-docs/**,\
-  ${server.servlet.path}/issuance/**
+  ##---------------------------------Vehicle Insurance Credential Configurations------------------------------------------------------
 
-mosip.certify.security.ignore-auth-urls=/actuator/**,**/error,**/swagger-ui/**,\
-  **/v3/api-docs/**, **/issuance/**
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/vehicle-insurance-credential.json
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.cred-schema-id=did:schema:e95a24f5-d65e-4b65-b497-e0671f8bd12d
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.cred-schema-version=1.0.0
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
```

Comment:
Do we need VehicleInsuranceCredential ? Is Life and Health Insurance are not our defaults ?

Reply:
removed VehicleInsuranceCredential